packages/react-ui/src/types.d.ts

```diff
@@ -225,13 +225,10 @@ export interface SelectFieldProps extends TextFieldProps {
   size?: 'small' | 'medium';
 }
 
+// UploadField
 export interface UploadFieldProps extends TextFieldProps {
-  placeholder?: string;
-  dragPlaceholder?: string;
   buttonText?: string;
   accept?: string[];
-  multiple?: boolean;
```

Comment:
Same comment as for propTypes: `multiple` is still used in the component, so shouldn't be removed here.